src/jit/compiler.h

```diff
@@ -281,17 +281,17 @@ class  LclVarDsc
     unsigned char       lvCustomLayout      :1;  // True when this struct has "CustomLayout"
     unsigned char       lvIsMultiRegArgOrRet:1; // Is this a struct that would be passed or returned in multiple registers?
 
-#ifdef _TARGET_ARM_
-    // TODO-Cleanup: Can this be subsumed by the above?
-    unsigned char       lvIsHfaRegArg:1;        // Is this argument variable holding a HFA register argument.
-    unsigned char       lvHfaTypeIsFloat:1;     // Is the HFA type float or double?
-#endif // _TARGET_ARM_
+#ifdef FEATURE_HFA
+    unsigned char       _lvIsHfa         : 1;   // Is this a struct variable who's class handle is an HFA type
+    unsigned char       lvIsHfaRegArg    : 1;   // Is this a HFA argument variable
```

Comment:
Is lvIsHfaRegArg needed now? because we already have lvIsRegArg and that in conjunction with _lvIsHfa would alllow to test the same thing that lvIsHfaRegArg is standing for.